control_routing_faulting_scripting

```diff
@@ -12,5 +12,5 @@ XB-UserVisible: yes
 Section: Add-Ons
 XB-DisplayVersion: {display_version}
 XB-DisplayName: NI Routing and Faulting LabVIEW Support for VeriStand {veristand_version}
-Depends: {AUTOVERSION_ni-labview-{labview_version}-x86}, {ni-veristand-{labview_version}} (>= {labview_short_version}.0.0)
+Depends: {AUTOVERSION_ni-labview-{labview_version}-x86}, {ni-veristand-{labview_version}} (>= {labview_short_version}.0.0), ni-veristand-{veristand_version}-custom-device-labview-support-common (>= {labview_short_version}.0.0)
```

Comment:
@buckd what should we be using for the required version of ni-veristand-{veristand_version}-custom-device-labview-support-common?

Reply:
`labview_short_version` is probably ok since it will have to rev for every new major version of LabVIEW, which should correspond to a release version of the custom devices/labview-support-common.

Reply:
After further testing, `labview_short_version` won't work properly. That will vary per version of LabVIEW, not per version of the custom devices, which means upgrades won't happen as expected for older VeriStand versions.

Reply:
Confirmed that things behave reasonably if we omit the version entirely.